### ISSUE TITLE
Sorting indicator not "activated" for default sorted column

### ISSUE THREAD

**Issue by thomasdsouza** [ISchwarz23/SortableTableView#37]
Hello Ingo, Thank you for such a great contribution!

Can you please let me know how to "activate" the sorting indicator for the default sorted column? My goal is to clearly communicate to the user which column is sorted and in which direction. Once the user clicks on a particular column, the indicator is clearly activating but not before clicking.
thanks again!

**Comment by ISchwarz23** [comment]
Hi Thomas,

Thanks for the nice words. Could you please state which method you are using for the "default" sorting column?

Best regards,
Ingo

**Comment by thomasdsouza** [comment]
Hi Ingo, I could use SortableTableView.sort() but really even if i didn't specifically call any sort method, I would still prefer the view to show which column is sorted to start off with (i.e. default if you did nothing but just followed the sample setup). Thanks much!

**Comment by ISchwarz23** [comment]
If you do not call the method `void sort(final int columnIndex)` or `void sort(final int columnIndex, final boolean sortUp)` or `void sort(final Comparator<T> comparator)` it is not sorted at all. It is simply using the order of the given data. This is the reason why no sorting indicator is shown.  
If you want to have the same "default" sorting for a certain table in your app, I recommend using the method `void sort(final int columnIndex, final boolean sortUp)`.

**Comment by thomasdsouza** [comment]
Hi Ingo, thanks for your response. Even if i sort by calling any of the sort( ) functions, data is being sorted but indicator is not being shown.

**Comment by ISchwarz23** [comment]
I can't confirm that. I experienced the expected behaviour which is:
- `sort(Comparator<T> comparator)` -> sorts the column using the given comparator and will not display any indicator (as the given comparator does not need to belong to a certain column)
- `sort(int index)` -> sorts the column ascending using the comparator set for the column with the given index. A second call to this method with the same column index will sort the table descending using the comparator set for the column with the given index. (behaves like a click on the column header)
- `sort(int index, boolean sortUp)` -> sorts the table ascending or descending depending on the sortUp param using the comparator set for the column with the given index *(suitable for creating a programatic "default" sorting)*  

I have tested this with the currently latest version (2.0.1)
